### PR TITLE
Run reducer optional

### PR DESCRIPTION
**Pull request by gsriram7**

### CODE REVIEW

**Comment by gsriram7** [comment]
@ashwanthkumar removed PailDuplicateWeeks job and used PosixParser instead of scalding-args to parse the args

**Comment by ashwanthkumar** [comment]
@gsriram7 Please help me understand. The only reason we use Hadoop MR job instead of a scalding job is the scalding dependency?

**Comment by ashwanthkumar** [comment]
Another point - Do we even need to support running reducers? Given it has not given us any value ? Also do you think we need the blacklist support too? It was written for a very specific use-case during a migration. I would really like to remove redundant code if possible.

**Comment by gsriram7** [comment]
@ashwanthkumar We use Hadoop MR because we didn't want to introduce scalding dep. Yes I ll remove reducers completely its of no use. I left blacklisting code thinking it might be useful someday.

**Comment by ashwanthkumar** [comment]
@gsriram7 
>  I left blacklisting code thinking it might be useful someday.

We'll add it if we need it? also it needs a code change anyways. Thoughts?

File: dfs-datastores/src/main/scala/com/indix/pail/PailMigrate.scala [hunk 1]
```diff
@@ -45,7 +44,7 @@ class PailMigrate extends Tool with ArgsParser {
     val recordType = cmdArgs("record-type")
     val recordClass = Class.forName(recordType)
 
-    val keepSourceFiles = cmdArgs("keep-source").toBoolean
+    val keepSourceFiles = cmdOptionalArgs("keep-source").getOrElse("false").toBoolean
```
Comment (ashwanthkumar) [inline dfs-datastores/src/main/scala/com/indix/pail/PailMigrate.scala]: Selva, can we please keep this a mandatory argument? That way people are cognizant when they want to delete stuff or not.

**Pull request merged**